src/views/scans/viewScansList.tsx

```diff
@@ -255,7 +255,9 @@ const ScansListView: React.FunctionComponent = () => {
                         label: t('table.label', { context: 'download' }),
                         disabled: !helpers.canDownloadReport(scan?.most_recent),
                         onClick: () => {
-                          downloadReport(scan.most_recent.report_id);
+                          if (scan?.most_recent) {
```

Comment:
I had to add this condition so typescript stop complaining, but it seems pointless given the button will be disabled if `scan.most_recent` is `undefined`. 

Is there a better way to handle these situations, @cdcabrera?

Reply:
cool. thanks for the alternatives!